cmd/docker-app/split.go

```diff
@@ -17,7 +17,7 @@ func splitCmd() *cobra.Command {
 		Short: "Split a single-file application into multiple files",
 		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			extractedApp, err := packager.ExtractWithOrigin(firstOrEmpty(args))
+			extractedApp, err := packager.Extract(firstOrEmpty(args))
```

Comment:
I'm maybe missing something but that's the only place we don't cleanup.

Reply:
Good point, I'll update :wink:

Reply:
nvw, we do cleanup :stuck_out_tongue_closed_eyes: